SALib/util/__init__.py

```diff
@@ -280,3 +280,70 @@ def _wrapper(*args, **kwargs):
             return result
         return _wrapper
     return _outer_wrapper
+
+
+def limit_samples(samples, upper_bound, lower_bound, dist):
+    '''
+    limits the array of samples passed as first arguments by replacing
+    the values falling outside the range [upper_bound lower_bound] with
+    the bounds themselves
+
+       Arguments
+       ---------
+       samples : ndarray
+           array containing the samples to be limited
+       upper_bound : float
+           the upper bound samples values will be limited to
+       lower_bound : float
+           the lower bound samples values will be limited to
+       dist: list
+           a list of the distributions the samples will be non uniformely scaled to
+       Returns
+       -------
+       limited_samples : ndarray
+           array containing the limited samples
+    '''
+    bounds = []
+    for i in dist:
+        if i in ['norm', 'lognorm']:
+            bounds.append([lower_bound,upper_bound])
+        else:
+            bounds.append([0,1])
+
+    limited_sample = samples
+    scale_samples(samples,bounds)
+
+    return limited_sample
+
+def checkBounds(problem):
```

Comment:
To comply with python style guides, camel case should not be used please use lower case and underscores for function names.